### PR TITLE
updated support page

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Review by jellecumulocity — APPROVED** [review]
Support page is good like this. Note that the other pages under 'additional resources' still need new references for downloads and Tech Community.

**Comment by BeateRixen** [comment]
> Support page is good like this. Note that the other pages under 'additional resources' still need new references for downloads and Tech Community.
@jellecumulocity Thanks, Jelle. Yes, we will replace everything step by step as soon as it is available. Can we already merge this PR and link to the new Support URL? Or when is the official date when we switch to it?

**Comment by jellecumulocity** [comment]
> > Support page is good like this. Note that the other pages under 'additional resources' still need new references for downloads and Tech Community.
> > @jellecumulocity Thanks, Jelle. Yes, we will replace everything step by step as soon as it is available. Can we already merge this PR and link to the new Support URL? Or when is the official date when we switch to it?

Let's wait with the merge until we have go-live confirmed at least. I don't mind having it available on c8y.com/docs before cutover, but not too long.
Also, I'm being asked by Product Management to change the name from Support Portal to something else. Customer Service Desk perhaps...

**Comment by BeateRixen** [comment]
> > > Support page is good like this. Note that the other pages under 'additional resources' still need new references for downloads and Tech Community.
> > > @jellecumulocity Thanks, Jelle. Yes, we will replace everything step by step as soon as it is available. Can we already merge this PR and link to the new Support URL? Or when is the official date when we switch to it?
> 
> Let's wait with the merge until we have go-live confirmed at least. I don't mind having it available on c8y.com/docs before cutover, but not too long. Also, I'm being asked by Product Management to change the name from Support Portal to something else. Customer Service Desk perhaps...

@jellecumulocity Please let me know if any naming change is required and when these changes can go live.

**Review by MWindel — APPROVED** [review]

**Comment by BeateRixen** [comment]
@jellecumulocity  What is the (final) URL now that I can use in the documentation? Is it still https://cumulocity.atlassian.net/servicedesk/customer/? And is the new portal already launched so that I can merge the PR? And how about the naming change you mentioned above (Support Portal)?

**Comment by jellecumulocity** [comment]
> @jellecumulocity What is the (final) URL now that I can use in the documentation? Is it still https://cumulocity.atlassian.net/servicedesk/customer/? And is the new portal already launched so that I can merge the PR? And how about the naming change you mentioned above (Support Portal)?

That is technically still the url, but we managed to get https://cumulocity.com/support pointing there. Please use that one in the docs.

**Comment by BeateRixen** [comment]
> > @jellecumulocity What is the (final) URL now that I can use in the documentation? Is it still https://cumulocity.atlassian.net/servicedesk/customer/? And is the new portal already launched so that I can merge the PR? And how about the naming change you mentioned above (Support Portal)?
> 
> That is technically still the url, but we managed to get https://cumulocity.com/support pointing there. Please use that one in the docs.

@jellecumulocity How about my other 2 questions :-)?

File: themes/c8ydocs/layouts/shortcodes/c8y-support-link.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{{- "https://cumulocity.atlassian.net/servicedesk/customer/" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/c8y-support-link.html]:
```suggestion
{{- "https://cumulocity.com/support" -}}
```

**Comment by jellecumulocity** [comment]
> @jellecumulocity How about my other 2 questions :-)?

Oops, sorry. It's not launched yet; hoping to do that later this week.
And the name is indeed [Cumulocity] Customer Service Desk.

File: themes/c8ydocs/layouts/shortcodes/c8y-support.html [hunk 2]
```diff
@@ -0,0 +1 @@
+{{- "Cumulocity Support Portal" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/c8y-support.html]:
```suggestion
{{- "Cumulocity Customer Service Desk" -}}
```

**Comment by BeateRixen** [comment]
@jellecumulocity Thanks! I updated the variables accordingly. Please let me know when we should publish these changes.

File: content/additional-resources/contacting-support.md [hunk 3]
```diff
@@ -6,16 +6,15 @@ sector:
   - getting_started
 ---
 
-Product support for {{< product-c8y-iot >}} is provided to licensed customers via our [{{< sag-portal >}}]({{< link-sag-portal >}}). Many services on this portal require that you have an account. If you do not yet have one, you can request it at [{{< link-sag-portal >}}register]({{< link-sag-portal >}}register).
+Product support for {{< product-c8y-iot >}} is provided to licensed customers via the [{{< c8y-support-portal >}}]({{< c8y-support-link >}}). The services on this portal require that you have an account. If you do not yet have one, you can send a request by email to [{{< c8y-support-email >}}]({{< c8y-support-email >}}).
 
-Once you have an account, you can, for example:
+Once you have an account, and depending on role-associated permissions, you can:
```
Comment (BeateRixen) [inline content/additional-resources/contacting-support.md]:
```suggestion
Once you have an account, and depending on role-associated permissions, you have the following options:
```

File: content/additional-resources/contacting-support.md [hunk 4]
```diff
@@ -6,16 +6,15 @@ sector:
...
 
-Download products, updates, and fixes.
-Search the Knowledge Center for technical information and tips.
-Subscribe to early warnings and critical alerts.
-Open and update support incidents.
-Add product feature requests.
+- Download products.
```
Comment (BeateRixen) [inline content/additional-resources/contacting-support.md]:
```suggestion
- ** Support**: Access technical assistance, documentation, and submit support tickets.
- **Feedback**: Share your thoughts and suggestions to help improve features and user experience.
- **Managed Services**: Access premium platform management, monitoring solutions, and specialized support.
```

File: content/additional-resources/contacting-support.md [hunk 5]
```diff
@@ -6,16 +6,15 @@ sector:
...
+- Search the Knowledge Base for articles.
```
Comment (BeateRixen) [inline content/additional-resources/contacting-support.md]:
```suggestion
```

File: content/additional-resources/contacting-support.md [hunk 6]
```diff
@@ -6,16 +6,15 @@ sector:
...
+- Open and update support tickets.
```
Comment (BeateRixen) [inline content/additional-resources/contacting-support.md]:
```suggestion
```

File: content/additional-resources/contacting-support.md [hunk 7]
```diff
@@ -6,16 +6,15 @@ sector:
...
+- Share product feedback.
```
Comment (BeateRixen) [inline content/additional-resources/contacting-support.md]:
```suggestion
```

File: themes/c8ydocs/layouts/shortcodes/c8y-support-portal.html [hunk 8]
```diff
@@ -0,0 +1 @@
+{{- "Cumulocity Support Portal" -}}
```
Comment (BeateRixen) [inline themes/c8ydocs/layouts/shortcodes/c8y-support-portal.html]:
```suggestion
{{- "Cumulocity Customer Service Desk" -}}
```

Comment (jellecumulocity) [inline themes/c8ydocs/layouts/shortcodes/c8y-support-portal.html]:
```suggestion
{{- "Cumulocity Support Portal" -}}
```
Customer Service Desk

**Review by jellecumulocity — APPROVED** [review]

**Pull request merged**